package.xml

```diff
@@ -16,6 +16,7 @@
   <url type="bugtracker">https://github.com/ros2/teleop_twist_keyboard/issues</url>
 
   <exec_depend>rclpy</exec_depend>
+  <exec_depend>geometry_msgs</exec_depend>
```

Comment:
oh my bad it was already upstream, just got confused by the commit title I guess. Sorry for the noise